bin/main.rb

```diff
@@ -22,3 +22,21 @@
 ]
 
 puts Oppen.print tokens: list, line_width: 25
+
+puts '--------------------------------------'
+
+wadler = Oppen::Wadler.new(line_width: 25)
+
+wadler.nest {
+  wadler.text 'XXXXXXXXXX'
+  wadler.breakable
+  wadler.text '+'
+  wadler.breakable
+  wadler.text 'YYYYYYYYYY'
+  wadler.breakable
+  wadler.text '+'
+  wadler.breakable
+  wadler.text 'ZZZZZZZZZZ'
+}
+
+puts wadler.output
```

Comment:
Create a ticket to cleanup the main.rb and remove the reference to it from the gemspec. We want to keep it clean because it's a lib not a lib + a bin. We'll keep it for ourselves to test and experiment locally.